### ISSUE TITLE
Add wallet to remove dependency on metamask

### ISSUE THREAD

**Issue by ayushdubey003** [NITJSR-OSS/MemeCash#15]

**Comment by Yash-R** [comment]
@ayushdubey003 please assign me this task

**Comment by Yash-R** [comment]
Also can you give me some idea how should i approach to this issue

**Comment by Yash-R** [comment]
Okay i got it, So are you planning to make a wallet in this application which will show all the transactions and information of MetaMask account @ayushdubey003 ??

**Comment by ayushdubey003** [comment]
Not Exactly. I plan to handle the transactions from the web app itself (through the wallet) rather than using a 3rd party extension like metamask

**Comment by Yash-R** [comment]
Yeah, that's what i'm saying you want the wallet in the web app for transactions right?

**Comment by Yash-R** [comment]
if you are planning to make a wallet inside the web app we can create a React Dapp like this one by web3.js

![Screenshot (13)](https://user-images.githubusercontent.com/41328269/97774071-e06afc80-1b7a-11eb-9396-a7773f9fb7b3.png)

**Comment by ayushdubey003** [comment]
Actually the thing is this is already implemented in the Dapp but for processing transactions (like signing transactions) we require 3rd party applications like metamask. So we want to remove this dependency and for that purpose we require it although it's just an enhancement.

**Comment by Yash-R** [comment]
Okay, So the Dapp is your another project because i didn't found in this web app.

**Comment by ayushdubey003** [comment]
The dapp is this project. You can see the readme for the url and instructions

**Comment by Yash-R** [comment]
Got it now, The Idea is actually very good im new in blockchain and learned basics of blockchain through this project can you tell me the resources or any thing which can give me more info and motivation to learn more or anything from where you have learned please forward to me and if you want any help for collaboration i can help and try to do by researching !

**Comment by ayushdubey003** [comment]
Thanks!
It was kind of just YouTube tutorials (Dapp University (You might have come across it)) and some random articles
